react-app/package.json

```
@@ -25,7 +25,7 @@
     "@mui/material": "5.16.0",
     "@mui/x-data-grid": "7.12.0",
     "@mui/x-date-pickers": "7.12.0",
-    "@turf/turf": "7.0.0",
+    "@turf/turf": "7.1.0",
```

```suggestion
    "@turf/turf": "7.0.0",
```

I think this change is in your other pr?